### DISCUSSION TITLE
Upload folder/file in user folder

### DISCUSSION THREAD

**Discussion by FreeWind6** [immich-app/immich#14823] [General]
How will Immich react if I copy my photo folders to the user folder?
For example: I will copy folder "folder1" to folder /upload/{uuid-user}

**Comment by bo0tzz** [comment]
It will do nothing. You shouldn't mess with the files inside UPLOAD_LOCATION (just reading them is fine).

**Reply by FreeWind6** [reply]
thanks

**Comment by FreeWind6** [comment]
PS, i`m not use "Storage Template"
